docs/weblog/README.md

```diff
@@ -746,3 +752,13 @@ Query parameters required in the `GET` method:
 
 Examples:
 - `GET`: `/session/user?sdk_user=sdkUser`
+
+### \[GET\] /user/me
```

Comment:
not sure what's the difference between this endpoint and `/user_login_success_event` or `/login` ?

Reply:
I'll clarify it in the documentation, this endpoint should be called once the user has already been logged into the system (meaning no auto login event generated).

Reply:
This endpoint was originally designed to test session fingerprinting that required a user ID, the fact that this is under the `/session` path, makes it clear that it requires an active session (but we could have reused the ones that you mentioned ofc). From the doc:

> Once a session has been established, a new call to `/session/user` must be made in order to generate a session fingerprint with the session id provided by the web client (e.g. cookie) and the user id provided as a parameter.

Reply:
idk it feels like it's an additional endpoint that is not very different from the ones we already have and we could use those instead, it's a bit confusing imo why we'd need this one

Reply:
Removed, the new version does not create new endpoints (it uses `/` instead)